atomate/vasp/firetasks/electrode_tasks.py

```diff
@@ -1,4 +1,5 @@
 import math
+from pydash import get
```

Comment:
I don't think you need this import?

Reply:
My mistake! I removed this import statement.